lib/services/events/analyzers/google-analytics.js

```diff
@@ -18,6 +18,7 @@ module.exports = options => {
 
 Analyzer.prototype = {
 	prepare(payload) {
+		payload = payload.attributes;
```

Comment:
`.attributes` should not be used in this context since its assuming the JSON API Service is being used which it could not be if someone doesn't want to.